doc/styles/TextBox.md

```diff
@@ -15,75 +15,75 @@ IsDefaultStyle\*: Styles in this column will be set as the default implicit styl
+| `OutlinedTextBoxPlaceholderForegroundFocused`     | `SolidColorBrush` | `OnSurfaceBrush`              |
+| `OutlinedTextBoxPlaceholderForegroundDisabled`    | `SolidColorBrush` | `OnSurfaceLowBrush`           |
+| `OutlinedTextBoxHeaderForeground`                 | `SolidColorBrush` | `OnSurfaceVariantBrush`       |
+| `OutlinedTextBoxHeaderForegroundPointerOver`      | `SolidColorBrush` | `OnSurfaceVariantBrush`       |
+| `OutlinedTextBoxHeaderForegroundFocused`          | `SolidColorBrush` | `PrimaryBrush`                |
+| `OutlinedTextBoxHeaderForegroundDisabled`         | `SolidColorBrush` | `OnSurfaceLowBrush`           |
+| `OutlinedTextBoxFontFamily`                       | `FontFamily`      | `MaterialMediumFontFamily`    |
+| `OutlinedTextBoxFontWeight`                       | `String`          | `BodyLargeFontWeight`         |
+| `OutlinedTextBoxFontSize`                         | `Double`          | `BodyLargeFontSize`           |
+| `OutlinedTextBoxCharacterSpacing`                 | `Int32`           | `BodyLargeCharacterSpacing`   |
+| `FilledTextBoxFontFamily`                         | `FontFamily`      | `MaterialMediumFontFamily`    |
+| `FilledTextBoxFontWeight`                         | `String`          | `BodyLargeFontWeight`         |
+| `FilledTextBoxFontSize`                           | `Double`          | `BodyLargeFontSize`           |
+| `FilledTextBoxCharacterSpacing`                   | `Int32`           | `BodyLargeCharacterSpacing`   |
+| `FilledTextBoxBorderThicknessNormal`              | `Double`          | `TextBoxOutlinedStrokeHeight` |
+| `FilledTextBoxBorderThicknessFocused`             | `Double`          | `TextBoxFocusStrokeWidth`     |
+| `TextBoxClearGlyphWidth`                          | `Double`          | 20                            |
+| `TextBoxClearGlyphHeight`                         | `Double`          | 20                            |
+| `FilledTextBoxCornerRadius`                       | `CornerRadius`    | 4,4,0,0                       |
+| `FilledTextBoxPadding`                            | `Thickness`       | 16,7                          |
+| `FilledTextBoxMinHeight`                          | `Double`          | 58                            |
+| `FilledTextBoxBorderHeightFocused`                | `Double`          | 2                             |
+| `OutlinedTextBoxBorderThickness`                  | `Double`          | 1                             |
+| `OutlinedTextBoxCornerRadius`                     | `CornerRadius`    | 4                             |
+| `OutlinedTextBoxPadding`                          | `Thickness`       | 8                             |
+| `OutlinedTextBoxMinHeight`                        | `Double`          | 56                            |
+| `OutlinedTextBoxBorderThicknessFocused`           | `Double`          | 2                             |
+| `OutlinedTextBoxBorderThicknessPointerOver`       | `Double`          | 2                             |
```

Comment:
@Xiaoy312, @kazo0 I think the breaking changes for lightweight styling should be also mentioned in the migration documentation right ? But not sure of the related version, next stable one?
https://github.com/unoplatform/Uno.Themes/blob/master/doc/material-migration.md